examples/custom-client-cxx/cli.cpp

```diff
@@ -0,0 +1,232 @@
+#include <iostream>
+#include <thread>
+
+#include <boost/log/core.hpp>
+#include <boost/log/expressions.hpp>
+#include <boost/log/trivial.hpp>
+#include <boost/process.hpp>
+
+#include "aktualizr-lite/api.h"
+#include "aktualizr-lite/cli/cli.h"
+#include "cli.h"
+
+#define LOG_INFO BOOST_LOG_TRIVIAL(info)
+#define LOG_WARNING BOOST_LOG_TRIVIAL(warning)
+#define LOG_ERROR BOOST_LOG_TRIVIAL(error)
+
+static void printStatus(aklite::cli::StatusCode ret) {
+  switch (ret) {
+    case aklite::cli::StatusCode::Ok:
+      std::cout << "SUCCESS";
+      break;
+
+    // Possible return codes for check, pull and install commands
+    case aklite::cli::StatusCode::CheckinOkCached:
+      std::cout << "SUCCESS: TUF data already up-to-date";
```

Comment:
Not sure this is correct message since `CheckinOkCached` is usually returned when a client fails to fetch meta from remote tuf repo. This code just states that a local TUF repo is valid.